sematic/ui/src/pipelines/PipelinePanels.tsx

```diff
@@ -41,10 +41,16 @@ export default function PipelinePanels() {
     return graph.runsById.get(runId) || rootRun;
   }, [selectedRunId, graph, rootRun]);
 
+  const defaultTab = selectedRun?.future_state === "FAILED" ? "logs" : "output";
```

Comment:
I think this might not do what you intended to do. Because initially, `selectedRun` is likely `undefined`. So on L45, the default value for `selectedRunTab` is fixated to `output`.

It's not a big deal, let's discuss what the right behavior should be, and we can fix it later.